### PR TITLE
Feature/urgency

### LINKED ISSUE #28: Implement front-end urgency service

### PR DESCRIPTION
**Pull request by cassiebaer**
closes #28

### CODE REVIEW

File: client/urgency.service.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+module.exports = {
+  prioritizeTasks
```
Comment (trevorwhealy) [inline client/urgency.service.js:2]: prioritizeTasks: prioritizeTasks ?

Comment (cassiebaer) [inline client/urgency.service.js:2]: ES6 property value shorthands ftw!

Comment (trevorwhealy) [inline client/urgency.service.js:2]: Ahhh - sahhweet!

**Comment by trevorwhealy** [comment]
🚢

**Comment by danejordan** [comment]
🚢

**Pull request merged**